nemo/collections/tts/torch/data.py

```diff
@@ -516,19 +506,13 @@ def __getitem__(self, index):
         # Load alignment prior matrix if needed
         align_prior_matrix = None
         if AlignPriorMatrix in self.sup_data_types_set:
+            # formula derived after investigating depths of librosa
+            mel_len = audio.flatten().shape[-1] + 2 * int(self.n_fft / 2) - (self.n_fft - 1)
+            mel_len = math.ceil(mel_len / self.hop_length)
```

Comment:
is there any reason why to replace `mel_len = self.get_log_mel(audio).shape[2]` with manual operations?

Reply:
Now that `beta_binomial_prior_distribution` is optimized, `get_log_mel` might take the majority for time needed for computing alignment prior matrices (stft + matmul inside). I transfered this from a scratch notebook where this was a concern. Not entirely sure if it makes any difference in routine data loading but is still nice to have.

Could benchmark if needed.

Reply:
Can you make a note that this is equivalent to `self.get_log_mel(audio).shape[2]` in the comment?